modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/coercion/BaseTypeCheckExecutionTest.java

```diff
@@ -134,6 +142,82 @@ private static Object generateConstantValueByType(NativeType type) {
         return generateConstantValueByType(type, "2");
     }
 
+    private static Object generateNotZeroValueByType(NativeType nativeType) {
```

Comment:
for the next time, it will be better to introduce some sort of strategies to generate different types of values.